### ISSUE TITLE
Unable to launch ROMs when built from source on OpenSUSE distro

### ISSUE THREAD

**Issue by biggestsonicfan** [dh4/cen64-qt#13]
As of commit 5f626ce34f1f3526b658d26d97a5a25d774a13f0, whenever a ROM is launched, all I see is either a blankly rendered X window or an X window with whatever was loaded behind it frozen into the window. For example:

![Screenshot_20200327_211723](https://user-images.githubusercontent.com/9744201/77814658-6583b580-7070-11ea-907a-c8cfef0042a8.png)

Cen64 when launched on it's own through the terminal runs ROM files just fine:

![Screenshot_20200327_211913](https://user-images.githubusercontent.com/9744201/77814691-babfc700-7070-11ea-9e6d-9d21e6006ad5.png)

If I terminate emulation early through Cen64-qt I recieve the following log output:
`/home/rob/Emulators/cen64/cen64 -eep4k "/home/rob/Emulators/ROMs/N64/N64 Saves/Super Mario 64 (U) [!].20b854b239203baf6c961b850a4a51a2.eep4k" -eep16k "/home/rob/Emulators/ROMs/N64/N64 Saves/Super Mario 64 (U) [!].20b854b239203baf6c961b850a4a51a2.eep16k" -sram "/home/rob/Emulators/ROMs/N64/N64 Saves/Super Mario 64 (U) [!].20b854b239203baf6c961b850a4a51a2.sram" -flash "/home/rob/Emulators/ROMs/N64/N64 Saves/Super Mario 64 (U) [!].20b854b239203baf6c961b850a4a51a2.flash" -controller num=1 "/home/rob/Emulators/BIOS/N64 BIOS/pifdata.bin" "/home/rob/Emulators/ROMs/N64/Super Mario 64 (U) [!].z64"`

I'm running OpenSUSE Tumbleweed and would love to help debug why the ROMs fail to load.

**Comment by dh4** [comment]
Did it work fine for you before that commit? That commit you referenced only changed the directory name of the temp directory where zipped ROMs are unzipped to, so I don't see how that commit would cause the regression you're describing since you're not using a zipped file.

I'll need to do some testing to see if I can reproduce what you're seeing.

**Comment by biggestsonicfan** [comment]
I apologize. I should not have referenced the commit. I merely included it to show the build I made for historical purposes. I had not used cen64-qt prior to this commit, so I am not accusing a commit that changed a directory name of a temporary directory for causing this issue.

I needed to reinstall my distro, so I had to recompile both cen64 and cen64-qt. The issue still remains, sadly. I will try going backwards to each commit to see where the issue could lie, however, the issue may be with OpenSUSE compatibility in general.

EDIT: I built commit 9d364a8493bad285f0685d43c7250354f20ddcd7 and there was no difference. unable to launch ROMs, merely a frozen image/blank image.
